packages/client/src/client/OpenFin/apps/Launcher/index.tsx

```diff
@@ -98,10 +98,10 @@ function Launcher() {
   useEffect(() => {
     setOverlay(overlayRef.current)
     getCurrentWindowBounds().then(setInitialBounds).catch(console.error)
-    registerCreditQuoteNotifications()
+    registerCreditNotifications()
```

Comment:
still need this (and Workspace one) for NLP stuff, so hard to de-dup this final additional registration on top of the `CreditRfqsCore` one

Reply:
this should really have the corresponding FX reg/unreg too, to be consistent

Reply:
Adding them increases the number of FX notification by a lot.

Reply:
just tested and do not get _any_ FX dups .. probs because it is triggering the notification send based off the FX Trade ACK, like Credit create/accept! ..
Yet again, it is the `quotes` that are dragging us back ...!